server/app/views/components/ProgramCardFactory.java

```diff
@@ -126,6 +121,11 @@ private DivTag renderProgramRow(
                 Styles.H_12,
                 programRow.extraRowActions().size() == 0 ? Styles.INVISIBLE : "");
 
+    PTag badge =
```

Comment:
The screenshots make it look like there's a fair amount of vertical spacing between the badge and the title. If that's what the mocks call for, then definitely ignore me.

Reply:
Yes, there is space between in the mocks.